view/frontend/templates/embed.phtml

```diff
@@ -51,11 +51,12 @@ use Nosto\Tagging\Block\Embed;
  * Please see a reference guide such as https://github.com/Nosto/nosto-magento2/wiki
  */
 
-/** @var Embed $block */
+/**
+ * @var Embed $block
+ * @var Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
+ */
 ?>
 <?php if ($block->getNostoScriptUrl()): ?>
   <!--suppress JSUnresolvedVariable -->
-  <script type="text/javascript"
```

Comment:
This should be fine as it is not an inline one, right?

Reply:
It was indeed not needed, I have reverted this change @olsi-qose